lib/settings.js

```diff
@@ -31,13 +31,15 @@ module.exports = async function settingsMiddleware (request, response, next) {
     updateCookie(response, SETTINGS_NAME, settings)
   }
 
-  // Allow enabling of editorial features via query parameters
-  const { enable_editorial_features } = request.query
-  if (enable_editorial_features !== undefined) { // eslint-disable-line camelcase
-    delete request.query.enable_editorial_features
-    settings.editorialFeatures = true
+  // Allow enabling and disabling of editorial features via query parameters
+  /* eslint-disable camelcase */
+  const { editorial_features } = request.query
+  if (editorial_features && ['enabled', 'disabled'].indexOf(editorial_features) !== -1) {
```

Comment:
remove ` && ['enabled', 'disabled'].indexOf(editorial_features) !== -1` and treat anything not `enabled` as disabled